packages/astro/README.md

```diff
@@ -28,4 +28,98 @@ This package is a wrapper around `@sentry/node` for the server and `@sentry/brow
 
 ## Installation and Setup
 
-TODO
+### 1. Registering the Sentry Astro integration:
+
+Add the `sentryAstro` integration to your `astro.config.mjs` file:
+
+```javascript
+import { sentryAstro } from "@sentry/astro/integration";
+
+export default defineConfig({
+  // Rest of your Astro project config
+  integrations: [
+    sentryAstro({
+      dsn: '__DSN__',
+    }),
+  ],
+})
+```
+
+This is the easiest way to configure Sentry in an Astro project.
+You can pass a few additional options to `sentryAstro` but the SDK comes preconfigured in an opinionated way.
+If you want to fully customize your SDK setup, you can do so, too:
+
+### 2. [Optional] Uploading Source Maps
+
+To upload source maps to Sentry, simply add the `project`, `org` and `authToken` options to `sentryAstro`:
```

Comment:
can we already get rid of the `org` with org auth tokens? 🤔 not 100% sure what the state there is TBH 😅

Reply:
Good point. I think we could, the thing that's still left is the wizard (which doesn't support Astro at the moment). 
I'll remove it from the manual setup documentation and if we create an Astro wizard, it can still add `org` if an old token (multi-org users) was created. 
I'll just test that this actually is supported by the vite plugin first 😅